driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/AggregatePublisherImpl.java

```diff
@@ -110,6 +111,12 @@
         return this;
     }
 
+    @Override
+    public AggregatePublisher<T> let(@Nullable final Bson variables) {
```

Comment:
[optional]
Here and in `AggregateIterableImpl.java` we specify `@Nullable` despite this class not being a part of the public API. Should we do the same in other non-public places where the `let` method was added?

Reply:
I do it here to match the public API that is being overridden.  I'm less inclined to do the same in driver-core, like AggregateOperation, esp. considering it's not done for any other properties.